src/platform/apollolake/include/platform/memory.h

```diff
@@ -303,7 +303,7 @@
 
 #define SOF_TEXT_START		(HP_SRAM_VECBASE_RESET + 0x400)
 #define SOF_TEXT_BASE		(SOF_TEXT_START)
-#define SOF_TEXT_MIN_SIZE	(0x19000 - 0x400)
+#define SOF_TEXT_MIN_SIZE	(0x19000 - 0x400 + 0x1000)
```

Comment:
We should use Kconfig here and only add if selected. Pls see @jajanusz for info. It's working well now.

Reply:
It works but is wasting memory, it's already explained in this comment - https://github.com/thesofproject/sof/pull/993#discussion_r255728659

Reply:
kconfig fixed